src/_includes/docs/swift-package-manager/migrate-objective-c-plugin.md

```diff
@@ -0,0 +1,344 @@
+Replace `plugin_name` throughout this guide with the name of your plugin.
+The below example uses `ios`, replace `ios` with `macos`/`darwin` as applicable.
+
+1. [Enable the Swift Package Manager feature][enableSPM].
+
+2. Start by creating a directory under the `ios`, `macos`, and/or `darwin`
+   directories.
+   Name this new directory the name of the platform package.
+
+   <pre>
+   /plugin_name/plugin_name_ios/ios/<b>plugin_name_ios</b>
```

Comment:
Note: a customer might not have the `plugin_name/plugin_name_ios/` directory structure. This is a commonly used pattern for federated plugins, but a user could also have a directory structure like `/plugin_name/ios/...`. Please let me know if you have suggestions to improve clarity.